### DISCUSSION TITLE
What is the difference between nuxt-graphql-client and your new nuxt module nuxt-apollo ?

### DISCUSSION THREAD

**Discussion by cfab** [Diizzayy/nuxt-graphql-client#225] [Q&A] [answered]
Hello, 
I just saw that you released/update for nuxt 3 the nuxt apollo module... 
I'm just wondering what are the differences with nuxt-graphql-client and in which use cases you would favor one module over the other.
Thank you

**Comment by Diizzayy** [comment, marked as answer]
@cfab The biggest differentiators for now are as follow:


| Feature | nuxt-graphql-client | @nuxtjs/apollo |
| ---------| ------------- | ------------- |
| Type Support | ✅ Full & Automatic  | ⚠️ Manual Typing  |
| Subscriptions | ❌ TODO (WIP)  | ✅ Supported |

I'd suggest `nuxt-graphql-client` for anyone looking for a more lightweight bundle and a better developer experience, where as `@nuxtjs/apollo` is the way to go if subscriptions are required, or if it's syntax is preferred.

**Reply by cfab** [reply]
Great thank you.

**Comment by awacode21** [comment]
i would like to ask the same question again in year 2024 @Diizzayy
